### ISSUE TITLE
Error Fetching with new Pokes

### ISSUE THREAD

**Issue by cesardgo** [billyvg/pokemon-journal#36]
If you have any of the new pre evolutions pokemons you will not be able to see data, error display is:

Error fetching inventory: TypeError: Cannot read property 'BaseStamina' of undefined

**Comment by billyvg** [comment]
Yeah https://github.com/billyvg/pokemon-journal/blob/master/app/api/baseStats.json needs to be updated to include the new pokemons

**Comment by cesardgo** [comment]
Thanks for the answer, but now o got this error: Cannot read property 'num' of undefined. Trying to fixit and nothing yet. 

I found on response.json the property num, and try to generate equal registry values for new pokemons but same error, will continue trying to fixit.

Thanks again And happy holidays.

**Comment by jefflembeck** [comment]
@cesardgo you'll also want to update https://github.com/billyvg/pokemon-journal/blob/master/app/api/pokemons.js as that is where the `num` for the `meta` attribute is pulled from.

Sidenote: do you have the baseStats.json updated? (because I'd totally love that info)

**Comment by cesardgo** [comment]
yes I update basestats.json and try to update pokemons.js with some like this (adding at the end of the file but the error continue, will download the original zip and trying again to update the files):


{
    "id":"239",
    "num":"239",
    "name":"Elekid",
    "img":"http://www.serebii.net/pokemongo/pokemon/239.png",
    "type": "Electric",
    "height": "0.41 m",
    "weight": "4.0 kg",
    "candy": "None",
    "egg": "10 km"
  }



________________________________
De: Jeff Lembeck <notifications@github.com>
Enviado: viernes 30 de diciembre de 2016 11:53 a.m.
Para: billyvg/pokemon-journal
Cc: cesardgo; Mention
Asunto: Re: [billyvg/pokemon-journal] Error Fetching with new Pokes (#36)


@cesardgo<https://github.com/cesardgo> you'll also want to update https://github.com/billyvg/pokemon-journal/blob/master/app/api/pokemons.js as that is where the num for the meta attribute is pulled from.

Sidenote: do you have the baseStats.json updated?

-

**Comment by jefflembeck** [comment]
@cesardgo here's my current pokemons.js file: https://github.com/jefflembeck/pokemon-journal/blob/71484172053bb2b478386b5481d8ac6d077c7df6/app/api/pokemons.js

I'd _love_ the new baseStats.json if you've got it (i'll make a PR to this repo once I have that)

**Comment by mgedwards** [comment]
Pull Request #37 created to add pokemon. Just the babies and togepi for now, but all the basestats are  updated now.

**Comment by aisbergde** [comment]
Is it possible to compile a windows version using the new pokemon and to make it available for download? This would be great! (In August I was not able to compile it myself)
